### PR TITLE
Doc/browser

### PR DESCRIPTION
**Pull request by mhewson**

### CODE REVIEW

File: cedar-data.json [hunk 1]
```diff
@@ -997,7 +997,13 @@
 								"type": "string",
 								"default": "\"p\"",
 								"description": "Valid HTML tag"
-							}
+							},
+							{
+							"name": "modifier",
+							"type": "string",
+							"default": "N/A",
+							"description": "Modifier allows the user to pass a style variant to this component  Possible values: body | display | display-static | heading-large | heading-large-static | heading-large-static | heading-medium | heading-medium-static | heading-small | heading-small-static | subheading"
```
Comment (KrisKnabel) [inline cedar-data.json]: "heading large static" possible value duplicated twice.

Comment (KrisKnabel) [inline cedar-data.json]: Remove duplicate possible value "heading large static"

File: docs/about-cedar/browser-support/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Cedar supports the latest, stable releases of the following browsers and platforms. Support is not guaranteed for platforms beyond this list, and any issues reported against unsupported platforms will be investigated at the team’s earliest opportunity.
+Officially supported platforms: are:
+
+| **Platform** | **Chrome** | **Firefox** | **Safari**     |
+| :----------- | :--------- | :---------- | :------------- |
+| Windows 10   | Yes        | Yes         |                |
+| Windows 7    | Yes        |             |                |
+| Mobile iOS   |            |             | Mobile version |
+| OS X 10.13   | Yes        |             | Yes            |
```
Comment (KrisKnabel) [inline docs/about-cedar/browser-support/README.md]: Are we really not supporting IE or Edge?

Comment (mhewson) [reply docs/about-cedar/browser-support/README.md]: thats a question for Test and Zack

**Pull request merged**